### ISSUE TITLE
Warning in cornerpooling for a model in pytorch loss.bacward()

### ISSUE THREAD

**Issue by icesohelrana** [bishwarup307/cornerpool#1]
Hi, I added cornerpooling in our pytorch model. But it bursts following warning. Do you know how to fix this issue?

[W Resize.cpp:19] Warning: An output with one or more elements was resized since it had shape [877], which does not match the required output shape [434].This behavior is deprecated, and in a future PyTorch release outputs will not be resized unless they have zero elements. You can explicitly reuse an out tensor t by resizing it, inplace, to zero elements with t.resize_(0). (function resize_output)
[W LegacyDefinitions.cpp:72] Warning: masked_scatter_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead. (function masked_scatter__cuda)
[W LegacyDefinitions.cpp:28] Warning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead. (function masked_fill__cuda)

**Comment by bishwarup307** [comment]
Hi, thanks for the issue. Can you give me a minimal example to reproduce this issue? Which python, pytorch and cuda version you are using?

**Comment by icesohelrana** [comment]
You can use this code to have a look.
`import torch.nn as nn`
`import torch`
`from cornerpool import TopPool`

`inp = torch.randn((1,3,10,10)).cuda()`

`conv=nn.Conv2d(3,1,3,padding=1).cuda()`
`toppool=TopPool().cuda()`

`out=conv(inp)`
`out=toppool(out)`
`target=torch.randn(out.shape).cuda()`
`criterion=torch.nn.SmoothL1Loss()`
`loss=criterion(out,target)`
`loss.backward()`

I'm using pytorch version 1.7.0 and torchvision version 0.8.1, Python version 3.7.9 and Cuda 10.2
